ccm_web/server/migrator.ts

```diff
@@ -0,0 +1,41 @@
+import { Umzug, SequelizeStorage } from 'umzug'
+import { Sequelize } from 'sequelize'
+import { validateConfig } from './src/config'
+import baseLogger from './src/logger'
+const logger = baseLogger.child({ filePath: __filename })
+
+const databaseConfig = validateConfig(process.env).db
+
+const sequelize = new Sequelize(
+  databaseConfig.name,
+  databaseConfig.user,
+  databaseConfig.password,
+  {
+    dialect: 'mysql',
+    host: databaseConfig.host
+  }
+)
+
+export const umzug = new Umzug({
+  migrations: {
+    glob: ['migrations/*.{js,ts}', { cwd: __dirname }],
```

Comment:
So in prod mode all the migration file compiled down to `js` so you have to tell umzug to look for `js` file as well, otherwise it will ignore it. for local development it is always `ts`.
if you look at the `SequelizeMeta` table the migration files will be indicated there. For local it will be `.ts` and prod `.js`.  I felt strange about this but really no difference in migrations being applied and tables created from that. I don't know the best practice here but the UMZUG author seems to support [this](https://github.com/sequelize/umzug#minimal-example). Look for little hidden accordion icon (I don't know how to call it) for typescript stuff

![Screen Shot 2021-05-20 at 9 55 34 AM](https://user-images.githubusercontent.com/8579775/118991343-a7954800-b951-11eb-8233-92677ba65380.png)

Reply:
Oh, duh, you are correct, thanks for pointing that out.